File: clean.py
import os

path = r'C:\Users\Windows11 Pro\.gemini\antigravity\scratch\electrical-audit-platform\index.html'
with open(path, 'r', encoding='utf-8') as f:
    content = f.read()

target = '<script src="app.js"></script>'
pos = content.find(target)
if pos != -1:
    new_content = content[:pos + len(target)] + '\n</body>\n</html>\n'
    with open(path, 'w', encoding='utf-8') as f:
        f.write(new_content)
    print("Successfully cleaned index.html")
else:
    print("Could not find script tag in index.html")

File: clean.js
const fs = require('fs');
const content = fs.readFileSync('index.html', 'utf8');
const lines = content.split(/\r?\n/);
const firstPart = lines.slice(0, 666).join('\n');
const lastPart = lines.slice(1371).join('\n');
const newContent = firstPart + '\n<script src="app.js"></script>\n' + lastPart;
fs.writeFileSync('index.html', newContent, 'utf8');
console.log('Successfully updated index.html');
